pipelines/_ingredients/LSSTCam-imSim/DRP.yaml

```diff
@@ -30,6 +31,10 @@ tasks:
       connections.newBFKernel: bfk
       doDefect: false
       doBrighterFatter: true
+      # Although we don't have to apply the amp offset corrections, we do want
+      # to compute them for analyzeAmpOffsetMetadata to report on as metrics.
+      doAmpOffset: true
+      ampOffset.doApplyAmpOffset: false
```

Comment:
Needed to add this here since t his YAML file does not load `_ingredients/DRP-full.yaml`.